xtask/src/license_headers_check.rs

```diff
@@ -312,6 +312,7 @@ lazy_static! {
         ("LICENSE\\..*", LicenseLocation::NoLicense),
         ("LICENSE-DejaVu", LicenseLocation::NoLicense),
         (".+\\.txt$", LicenseLocation::NoLicense),
+        (".+\\.x$", LicenseLocation::NoLicense),
```

Comment:
you date not having license for this file? :-)

Reply:
It's arguably small. But I guess I have to implement support for the comment style proper in the xtask. `c_style_comment_style` is actually C++ style `//`, while the linker script only supports C style (`/* .. */`).

Reply:
I decided not to implement C-style comments but keep the file as-is because it is not ours. I've added headers and changed the regex to match exactly the file.